target/linux/ramips/image/mt7621.mk

```diff
@@ -983,3 +983,20 @@ define Device/zio_freezio
 	kmod-usb-ledtrig-usbport wpad-basic
 endef
 TARGET_DEVICES += zio_freezio
+
+define Device/zyxel_wap6805
+  BLOCKSIZE := 128k
+  PAGESIZE := 2048
+  KERNEL_SIZE := 4096k
+  UBINIZE_OPTS := -E 5
+  IMAGE_SIZE := $(ralink_default_fw_size_32M)
+  DEVICE_VENDOR := ZyXEL
+  DEVICE_MODEL := WAP6805
+  DEVICE_PACKAGES := kmod-mt7603 wpad-basic kmod-mt7621-qtn-rgmii \
+	qv840-firmware qv840-utils
+  KERNEL += | mitrastarimage
```

Comment:
I'd prefer to have the full set of commands here, it's only a few and would match all the other devices.

Reply:
Assuming $(KERNEL_DTB) is acceptable as part of the command chain? Fixed

Reply:
Exactly.